app/Http/Transformers/ActionlogsTransformer.php

```diff
@@ -69,9 +71,36 @@ public function transformActionlog (Actionlog $actionlog, $settings = null)
 
                             if ($custom_field->db_column == $fieldname) {
 
-                                if ($custom_field->field_encrypted == '1') {
-                                    $clean_meta[$fieldname]['old'] = "************";
-                                    $clean_meta[$fieldname]['new'] = "************";
+                                if ($custom_field->field_encrypted == '1')  {
+
```

Comment:
This part seems solid - I'd agree, it'd be better to detect that the field didn't "really" change at the Observer level, but this does the trick and works back-in-time